pkg/agent/route/route_linux.go

```diff
@@ -418,6 +420,7 @@ func (c *Client) syncIPSet() error {
 		}
 	}
 
+	// Multicast is only available in encap mode.
```

Comment:
This is also wrong. Multicast supports noEncap.

Reply:
Got the answer from here https://github.com/antrea-io/antrea/blob/5c79f39c04bd5f98b493432389e00ea31ef616e0/pkg/agent/route/route_linux.go#L1836